doc: change the structure of doc files (#9561)
nrfconnect/sdk-nrf: doc/nrf/ecosystem_integration.rst

.. _ecosystem_integration:

Ecosystems integration
######################

> The following user guides describe the integration of various eco systems and tools with |NCS|.

Reviewer: ```suggestion
The following user guides describe the integration of various ecosystems and tools with |NCS|.
```

Author: fixed

The |NCS| is also integrated with the following ecosystems that are documented in private repositories:

* Apple Homekit

> * Apple FindMy

Reviewer 2: A
```suggestion
* Apple Find My
```

Reviewer 2: Not fixed yet.

Author: fixed

* ANT

> In the case of HomeKit and FindMy, MFi licensees can get access to the repository by issuing a Nordic `DevZone`_ private ticket.

Reviewer 2: ```suggestion
In the case of HomeKit and FindMy, MFi licensees can get access to the repository by issuing a Nordic `DevZone`_ private ticket.
```
```suggestion
In the case of HomeKit and Find My, MFi licensees can get access to the repository by issuing a Nordic `DevZone`_ private ticket.
```

Reviewer 2: Not fixed yet.

Author: fixed
